arch/PowerPC/PPCGenInstrInfo.inc

```diff
@@ -4050,18 +4050,18 @@ static const MCInstrDesc PPCInsts[] = {
   { 1, OperandInfo2 },
   { 0, nullptr },
   { 0, nullptr },
-  { 1, OperandInfo2 },
+  { 2, OperandInfo249 },
```

Comment:
why does this PR modify this array?

Reply:
i suppose this is because you run the syntool script on PPC td files?

Reply:
Yes.  I was able to use the sync scripts for a few of the backends, but not all.

Reply:
can you restore these parts (so this PR just focuses on adding const), so i can merge this first, and then we will come back to that later?